src/package-linker.js

```diff
@@ -380,6 +387,10 @@ export default class PackageLinker {
         linkBinConcurrency,
       );
     }
+
+    for (const [, {pkg}] of flatTree) {
+      await this._warnForMissingBundledDependencies(pkg);
```

Comment:
Why do we report these later on, instead of as we encounter above around line 112?